DigitalLearningSolutions.Data/Services/TrackerActionService.cs

```diff
@@ -0,0 +1,47 @@
+﻿namespace DigitalLearningSolutions.Data.Services
+{
+    using System.Linq;
+    using DigitalLearningSolutions.Data.DataServices;
+    using Newtonsoft.Json;
+    using Newtonsoft.Json.Serialization;
+
+    public interface ITrackerActionService
+    {
+        string GetObjectiveArray(int? customisationId, int? sectionId);
+    }
+
+    public class TrackerActionService : ITrackerActionService
+    {
+        private readonly JsonSerializerSettings settings;
+        private readonly ITutorialContentDataService tutorialContentDataService;
+
+        public TrackerActionService(ITutorialContentDataService tutorialContentDataService)
+        {
+            this.tutorialContentDataService = tutorialContentDataService;
+            settings = new JsonSerializerSettings { ContractResolver = new LowercaseContractResolver() };
+        }
+
+        public string GetObjectiveArray(int? customisationId, int? sectionId)

```

Comment:
I've had a thought - can we return a nullable c# type here rather than a JSON string? That makes testing this method a bit nicer, and will have the same effect for others. Then the JSON serialisation can go in TrackerService, just converting null to {}.